<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunSetupPartsDialog</class>
 <widget class="QDialog" name="RunSetupPartsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Setup Parts</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="main_area" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="instructions" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="line1">
           <property name="text">
            <string>Enter a value for each call argument of each Setup part. The parts are shown in execution priority.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="line2">
           <property name="text">
            <string>Click OK to run the parts with the given argument values, or Cancel to abandon this operation.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="arguments">
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="button_area" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="load_button">
        <property name="text">
         <string>Load...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="button_box_ok_cancel">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box_ok_cancel</sender>
   <signal>accepted()</signal>
   <receiver>RunSetupPartsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box_ok_cancel</sender>
   <signal>rejected()</signal>
   <receiver>RunSetupPartsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
